test/cleanups.test.ts

```diff
@@ -1,28 +1,17 @@
-import path from "node:path";
 import { describe, expect, it } from "vitest";
 
-import { runMutationTest } from "../src/tests/testSetup.js";
+import { checkTestResult } from "../src/tests/testSetup.js";
+
+const cwd = import.meta.dirname;
 
 describe("Cleanups", () => {
 	it("suppressTypeErrors", async () => {
-		const caseDir = path.join(
-			import.meta.dirname,
-			"cases/cleanups/suppressTypeErrors",
-		);
-		const { actualContent, expectedFilePath, options } =
-			await runMutationTest(caseDir);
-		await expect(actualContent).toMatchFileSnapshot(expectedFilePath);
-		expect(options).toMatchSnapshot("options");
+		expect.assertions(3);
```

Comment:
[Refactor] 😬 I really, _really_ dislike `expect.assertions`. It's super brittle: relying on implementation details cross-test. If you ever rework a test, add a utility that runs multiple assertions, etc. you end up having to change a bunch of otherwise-seemingly-arbitrary numbers. Plus it's not comprehensive - if some async logic gets added to tests and the author forgets to update a number, it can fail silently or in some future tests. Spooky.

Request: remove the `expect.assertions(...)` calls. If there are places that would need that logic, we can always look at them individually. (are there?)